vmaas_sync/vmaas_sync.go

```diff
@@ -0,0 +1,60 @@
+package vmaas_sync
+
+import (
+	"app/base/utils"
+	"github.com/gin-gonic/gin"
+	"github.com/gorilla/websocket"
+	ginprometheus "github.com/zsais/go-gin-prometheus"
+)
+
+func runMetrics() {
+	// create web app
+	app := gin.New()
+
+	prometheus := ginprometheus.NewPrometheus("gin")
```

Comment:
Use `patchman_engine` metric prefix instead of `gin`, please.

Reply:
Actually, there is middleware created to setup Prometheus metrics, please use it.

Reply:
Fixed.